RELEASES.md

```diff
@@ -1,3 +1,235 @@
+Version 1.83.0 (2024-11-28)
+==========================
+
+<a id="1.83.0-Language"></a>
+
+Language
+--------
+- [The `non_local_definitions` lint now uses a syntactic heuristic instead of type system based analysis](https://github.com/rust-lang/rust/pull/127117)
+- [atomics: allow atomic and non-atomic reads to race](https://github.com/rust-lang/rust/pull/128778)
+- [Non-exhaustive structs may be empty](https://github.com/rust-lang/rust/pull/128934/)
```

Comment:
> #128934 didn't have a relnotes tracking issue

Because it didn't have the actual relnotes label -- now it does and there's #133353.
(I think the relnotes tool picked it up by its FCP label.)